cmd/kueuectl/app/list/table_printer.go

```diff
@@ -0,0 +1,80 @@
+// Copied from https://github.com/kubernetes/kubectl/blob/a6de79e3d40575480caa71a8e5efffeea6587251/pkg/cmd/get/table_printer.go
```

Comment:
Why copy this instead of reusing it?

Reply:
Can you answer this question or reuse, instead of copying the code?

Reply:
Yeah, I think we can reuse. Will make the change.

Reply:
Done. https://github.com/kubernetes-sigs/kueue/pull/2280/commits/1020704040d15c91d568faddb073838f984cf780